### ISSUE TITLE
[Feature]: Chat auto translate using deepl API

### ISSUE THREAD

**Issue by sharkman** [LiveTL/HyperChat#114]
### Checklist

- [X] This is an issue for Hyperchat, not LiveTL. I will go to [the LiveTL repo](https://github.com/LiveTL/LiveTL) to report an issue for LiveTL.
- [X] I have checked for a duplicate or similar issue. I made sure to check closed issues as well.
- [X] I am not reporting a bug. If I am, I will use the bug report issue template.
- [X] By submitting this issue, you agree to follow our [Code of Conduct](https://github.com/LiveTL/HyperChat/blob/master/CODE_OF_CONDUCT.md)

### Is your feature request related to a problem? Please describe.

Google translate is pretty poor in translation quality in general regarding Asian languages.

### Describe the solution you'd like

You can use [Deepl API](https://www.deepl.com/pro-api) to auto translate in addition to Google translate. This may require an API token from user (not sure).

### Expected behavior

Translation quality should improve drastically.

**Comment by r2dev2** [comment]
Deepl is very expensive ($25.00 per 1,000,000 characters) so only way this gets implemented is if the user buys deepl and puts in an api key.

This seems very niche so if we do decide to implement this, it will be very low on the priority list.

**Comment by sharkman** [comment]
I agree, it is expensive and niche, that's why I use my Discord translation bot with the same functionality.
Unfortunately, it has one flaw: youtube API does not have access to live chat messages for the members-only streams (and they do not plan to implement this feature).
